### ISSUE TITLE
Issue with Input Parameters

### ISSUE THREAD

**Issue by SundaramMaheshkumar** [ralphhanna/bpmn-server#85]
Hi @ralphhanna,

_Using 1.3.15_

In the below workflow, there are two service tasks with their own input parameters.

![image](https://user-images.githubusercontent.com/118889895/230383587-f919d571-30d6-491b-b9b7-8152287ac4d5.png)

![image](https://user-images.githubusercontent.com/118889895/230383704-52843547-2de2-485b-b3ae-8760a0fe0839.png)

My Service:
```
async ServiceTask(input, context) {
        console.log("Inputs:", input);
}
```

Output:
![image](https://user-images.githubusercontent.com/118889895/230384572-43b77f53-5d96-4a71-8896-87c0cb1bddf2.png)

Although the input parameters `destination`  of the two service tasks are different; the console prints the same output for both the tasks!!!

Could you please look into this?

Thank you.

**Comment by ralphhanna** [comment]
You need to provide unique names or structure since all instance variables belong to the instance

**Comment by ralphhanna** [comment]
How are you invoking these services?
Can you please provide a complete sample code

**Comment by ralphhanna** [comment]
Fixed in Release 1.3.18
